frontend/pages/hosts/ManageHostsPage/ManageHostsPage.tsx

```diff
@@ -578,6 +583,23 @@ const ManageHostsPage = ({
     );
   };
 
+  const handleChangeBootstrapPackageStatusFilter = (
+    newStatus: BootstrapPackageStatus
+  ) => {
+    handleResetPageIndex();
+
+    router.replace(
+      getNextLocationPath({
+        pathPrefix: PATHS.MANAGE_HOSTS,
+        routeTemplate,
+        routeParams,
+        queryParams: Object.assign({}, queryParams, {
```

Comment:
Nit: The spread syntax would be a little more readable.

Reply:
updated